boards/qemu_rv32_virt/src/io.rs

```diff
@@ -8,7 +8,6 @@ use core::str;
 
 use kernel::debug;
 use kernel::debug::IoWrite;
-use rv32i;
```

Comment:
Weird this was caught in this particular lint, but w/e I guess

Reply:
Yeah it wasn't, clippy fix sometimes deletes a bunch of imports from some board's io.rs file. When I fixed this one I only put back what was necessary.

I don't really understand why io.rs and rust behave so strangely.